src/Raven.Server/Documents/Handlers/Debugging/MemoryStatsHandler.cs

```diff
@@ -136,14 +139,14 @@ private static DynamicJsonValue MemoryStatsInternal()
                 {
                     if (fileMappingByDir.TryGetValue(sizes.Key, out Dictionary<string, ConcurrentDictionary<IntPtr, long>> value))
                     {
-                        var details = new DynamicJsonValue();
-                        
                         var dir = new DynamicJsonValue
                         {
-                            [nameof(MemoryInfoMappingItem.Directory)] = sizes.Key.Substring(prefixLength),
-                            [nameof(MemoryInfoMappingItem.TotalDirectorySize)] = sizes.Value,
-                            [nameof(MemoryInfoMappingItem.HumaneTotalDirectorySize)] = Size.Humane(sizes.Value),
-                            [nameof(MemoryInfoMappingItem.Details)] = details
+                            ["Directory"] = sizes.Key.Substring(prefixLength),
```

Comment:
Why did you remove all `nameof()` usages? 
Why did you remove `MemoryInfo*` classes? 
Build does not compile: 
`00:06:07.719 Program.cs(351,56): error CS0426: The type name 'MemoryInfo' does not exist in the type 'MemoryStatsHandler' [c:\Jenkins-lite\workspace\PR_Compile_v4.0\s\tools\TypingsGenerator\TypingsGenerator.csproj]
`

Reply:
I didn't.. merge issue

Reply:
Did you have conflict? Please notice I've also changed format of endpoint output, to avoid mixing data with structure.